code/modules/clothing/under/marine_uniform.dm

```diff
@@ -517,12 +517,6 @@
 	worn_state = "pmc_jumpsuit"
 	min_cold_protection_temperature = ICE_PLANET_MIN_COLD_PROT
 	flags_jumpsuit = UNIFORM_SLEEVE_ROLLABLE
-	suit_restricted = list(
-		/obj/item/clothing/suit/storage/marine/veteran/pmc,
-		/obj/item/clothing/suit/storage/marine/smartgunner/veteran/pmc,
-		/obj/item/clothing/suit/armor/vest/security,
-		/obj/item/clothing/suit/storage/hazardvest,
-	)
```

Comment:
why tho?

Reply:
Because it literally wouldn't let you wear anything other than those few options. You couldn't wear a jacket over the pmc uniform, and I explained that in the (albeit very long) description